### PR TITLE
Adding bdw results

### PR DESCRIPTION
**Pull request by psteinb**
this is WIP as I am still waiting for some wisdom results

### CODE REVIEW

**Comment by psteinb** [comment]
ok, the wisdom based results are in now as well. this PR is ready for merge from my side.

File: results/broadwell/sbatch_furiosa_fftw_broadwell.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    srun --cpu-freq=medium $REL/gearshifft_fftw -f $FEXTENTS1DFFTW -o $RESULTS/fftw_estimate_gcc5.3.0_RHEL7.2.1d.csv --rigor estimate
+
+elif [ $k -eq 1 ]; then
+    srun --cpu-freq=medium $REL/gearshifft_fftw -f $FEXTENTS2D -o $RESULTS/fftw_estimate_gcc5.3.0_RHEL7.2.2d.csv --rigor estimate
+
+elif [ $k -eq 2 ]; then
+    srun --cpu-freq=medium $REL/gearshifft_fftw -f $FEXTENTS3D -o $RESULTS/fftw_estimate_gcc5.3.0_RHEL7.2.3d.csv --rigor estimate
+
+elif [ $k -eq 3 ]; then
+    srun --cpu-freq=medium $REL/gearshifft_fftw -f $FEXTENTS1DFFTW -o $RESULTS/fftw_wisdom_gcc5.3.0_RHEL7.2.1d.csv --rigor wisdom --wisdom_sp $CURDIR/results/broadwell/fftwf_wisdom_3.3.6-pl1_bdw.txt --wisdom_dp $CURDIR/results/broadwell/fftwf_wisdom_3.3.6-pl1_bdw.txt
```
Comment (tdd11235813) [inline results/broadwell/sbatch_furiosa_fftw_broadwell.sh]: `--wisdom_dp $CURDIR/results/broadwell/fftwf_wisdom_3.3.6-pl1_bdw.txt`
should be
`--wisdom_dp $CURDIR/results/broadwell/fftw_wisdom_3.3.6-pl1_bdw.txt`
same applies for 2d and 3d below.

**Comment by tdd11235813** [comment]
double precision results are missing in the fftw wisdom benchmarks. see comment.

other than that, we see the weird fftw behaviour, as it was discussed [in this issue](https://github.com/mpicbg-scicomp/gearshifft_publication/issues/54).
the broadwell used with 40 threads does not seem to be better than haswell with 24 threads, even when inspecting Time_FFT only. It is likely that fftw always used one core internally as for pure FFT computation the timescale is way too low (1us up to 1ms) to benefit from multi-core. The planning time is very high when using fftw-measure flag. Using only one core from the beginning could improve the planning time heavily without loosing FFT performance, see my post [here](https://github.com/mpicbg-scicomp/gearshifft_publication/issues/54#issuecomment-265703216).

edit: doing strong-scaling tests for fftw is not planned at the moment, but if time permits, i gonna add a single core fftw-measure result file to see the differences. For scaling tests the use of FFTWs `fftw-bench` is also recommended.

**Comment by psteinb** [comment]
I checked the jobs again and apparently the wisdom files are corrupted. I have to research why and how. In any case, I am regenerating them and will then rerun the examples again. Believe me, that comes to my surprise and I hope they will be done by mid this week.

**Comment by tdd11235813** [comment]
did you compile fftw and fftwf in the same directory? The tools might become confused, so it is more safe to compile to separate directories to have fftw_wisdom and fftwf_wisdom isolated.

**Comment by psteinb** [comment]
that is very likely the reason. thanks for the hint.

**Comment by tdd11235813** [comment]
as of issue #105 you might want to create a new PR on gearshifft_results to publish the broadwell measurements.

**Comment by psteinb** [comment]
will do.

**Pull request closed without merging**